easybuild/easyconfigs/r/Rivet/Rivet-3.1.9-gompi-2023a-HepMC3-3.2.6.eb

```
@@ -0,0 +1,52 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'Rivet'
+version = '3.1.9'
+local_hepmc = "HepMC3"
+local_hepmcver = "3.2.6"
+versionsuffix = '-%s-%s' % (local_hepmc, local_hepmcver)
+
+homepage = 'https://gitlab.com/hepcedar/rivet'
+description = """
+Rivet toolkit (Robust Independent Validation of Experiment and Theory)
+
+To use your own analysis you must append the path to `RIVET_ANALYSIS_PATH`.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2023a'}
+toolchainopts = {'optarch': True, 'pic': True}
```

`optarch` is enabled by default, so this should become:

```suggestion
toolchainopts = {'pic': True}
```